ij-plugin/src/main/resources/org/jetbrains/research/tasktracker/config/survey_default.yaml

```diff
@@ -1,210 +1,119 @@
 surveys:
   - id: default
     htmlQuestions:
-      - !<Input>
-        text: Approximately how many hours did you use the plugin?
-        elementId: hours
-        type: number
+      - !<Textarea>
+        text: 1.How helpful was the hint in solving the problem?
+        elementId: helpfulHint
         required: true
-        min: 0
-        step: 0.5
+        rows: 5
+        cols: 50
       - !<Radio>
-        text: 2. On a scale of 1 (very unlikely) to 5 (very likely), how likely are you to continue using the Emotion Tracker in the future?
-        elementId: likely
+        text: 2.Why did you decide to ask for a hint?
```

Comment:
For all questions with several options, we should add an ability for multi-choice questions